packages/api/src/platforms/vtex/resolvers/query.ts

```diff
@@ -335,4 +335,27 @@ export const Query = {
       sellers,
     }
   },
+  profile: async (_: unknown, { id }: QueryProfileArgs, ctx: Context) => {
+    const {
+      clients: { commerce },
+    } = ctx
+
+    const addresses = await commerce.profile.addresses(id)
```

Comment:
Do you think we should handle errors (try catch) if id didn't exist, for example? or did the Profile API already do?

Reply:
`id` here will always exist ensured by the query property that set it as required (see `QueryProfileArgs ` input type). If the param is not sent, the GraphQL layer will probably throw an error.

Reply:
That's right! The graphQl will throw an error 👍 
<img width="530" alt="image" src="https://github.com/user-attachments/assets/4995c062-b96d-4ae1-a974-ee905d984657" />